README.md

```diff
@@ -1,180 +1,30 @@
- 
-### To run the tests:
-Using Visual Studio:
-  - Build.
-  - "Test Explorer" window will get populated with tests. Select test and Run/Debug.
-
-Using the command line:
-msbuild .\build.proj /t:"Runtests" /p:Scope=SDKs\Compute
-in the above example RunTests will build and run tests for Compute only
-or
-dotnet test SDKs\Compute\Compute.Tests\Compute.Tests.csproj
-
-  - Refer to the "To build" section to get the command window set up.
-  - Invoke "RunTests" target from "Build.proj". RunTests will build and run tests 
-        *msbuild build.proj /t:RunTests /p:scope=SDKs\Compute*
-
-## To on-board new libraries
-
-### Project Structure
-
-In "SDKs\< Service Name >", you will find projects for services that have already been implemented
-
-  - Each SDK project needs to target .NET 4.5.2 and .NET Standard 1.4
-	  - Test project needs to target NetCoreApp 1.1
-  - Each service contains a project for their generated/customized code
-    - The folder 'Generated' contains the generated code
-    - The folder 'Customizations' contains additions to the generated code - this can include additions to the generated partial classes, or additional classes that augment the SDK or call the generated code
-    - The file 'generate.cmd', used to generate library code for the given service, can also be found in this project
-  - Services also contain a project for their tests
-
-### Standard Process
-
- 1. Create fork of [Azure REST API Specs](https://github.com/azure/azure-rest-api-specs)
- 2. Create fork of [Azure SDK for .NET](https://github.com/azure/azure-sdk-for-net)
- 3. Create your Swagger specification for your HTTP API. For more information see 
- [Introduction to Swagger - The World's Most Popular Framework for APIs](http://swagger.io)
- 4. Install the latest version of AutoRest and use it to generate your C# client. For more info on getting started with AutoRest, 
- see the [AutoRest repository](https://github.com/Azure/autorest)
- 5. Create a branch in your fork of Azure SDK for .NET and add your newly generated code to your project. If you don't have a project in the SDK yet, look at some of the existing projects and build one like the others. 
- 6. **MANDATORY**: Add or update tests for the newly generated code.
- 7. Once added to the Azure SDK for .NET, build your local package using command
- e.g.
- `msbuild build.proj /t:CreateNugetPackage /p:scope=SDKs\Compute`
- 8. If you're using **master** branch, bump up the package version in YourService.nuget.proj. If you're using **psSdkJson6** branch, change the package version in the .csproj file, as well as in the AssemblyInfo.cs file.
- 9.  A Pull request of your Azure SDK for .NET changes against **psSdkJson6** branch of the [Azure SDK for .NET](https://github.com/azure/azure-sdk-for-net)
- 11. Both the pull requests will be reviewed and merged by the Azure SDK team
+[Azure.Core](https://github.com/Azure/azure-sdk-for-net/blob/master/src/SDKs/Azure.Core/data-plane/README.md)
```

Comment:
@KrzysztofCwalina sorry for the late review. I know this is already merged, but could you tell me if the intention here is to highlight only data-plane related components, because it feels like this repo is only for data plane?
Any customer who might have visited this repo previously might feel that this repo is only for data plane.